user-guide/Getting_started/Installing_a_DMA/Unattended_DM_installation.md

```diff
@@ -4,6 +4,9 @@ uid: Unattended_DM_installation
 
 # Unattended DataMiner installation
 
+> [!IMPORTANT]
+> (Deprecated) This feature is not supported.
```

Comment:
@juliori85, I assume unattended installation is only possible with the older, deprecated installers? In that case, to make this more clear, I'd rephrase this as follows:
 ```suggestion
> This feature is deprecated. It is no longer available starting from version 10.4 of the DataMiner Installer.
```